pkg/graphdb/conn_sqlite3_dynamic_windows.go

```diff
@@ -0,0 +1,17 @@
+// +build cgo,windows,dynamicsqlite
+
+package graphdb
+
+import (
+	_ "code.google.com/p/gosqlite/sqlite3" // registers sqlite
+	"github.com/Sirupsen/logrus"
+)
+
+func checkDevBuild() {
+
+	//	dynamicsqlite should not be used for release builds. It is purely for
+	//	development purposes as it takes some 20 seconds less to build docker
+	//	on Windows with dynamic linking.
```

Comment:
You can forego the build time by doing `go build -i`. This will install mattn/go-sqlite3 into your `$GOPATH/pkg` and subsequent builds will not need to run cgo.